frontend/lint/eslint-rules/no-unconditional-metabase-links-render.js

```diff
@@ -0,0 +1,223 @@
+          );
+        }
+
+        // Named import
+        return variables.some(
+          variable =>
+            variable.defs[0].node.type === "ImportSpecifier" &&
+            variable.name === named,
+        );
+      }
+
+      return false;
+    }
+
+    return {
+      ImportDeclaration(node) {
+        if (
+          checkIfModuleIsImported(node, {
+            isDefault: true,
+            source: "metabase/lib/settings",
+          })
+        ) {
+          isMetabaseSettingsImported = true;
+        }
+        if (
+          checkIfModuleIsImported(node, {
+            named: "getDocsUrl",
+            source: "metabase/selectors/settings",
+          })
+        ) {
+          isGetDocsUrlSelectorImported = true;
+        }
+        if (
+          checkIfModuleIsImported(node, {
+            named: "getLearnUrl",
+            source: "metabase/selectors/settings",
+          })
+        ) {
+          isGetLearnUrlSelectorImported = true;
+        }
+        if (
+          checkIfModuleIsImported(node, {
+            named: "getShowMetabaseLinks",
+            source: "metabase/selectors/whitelabel",
+          })
+        ) {
+          isGetShowMetabaseLinksSelectorImported = true;
+        }
+      },
+      CallExpression(node) {
+        // call `getDocsUrl` selector
+        if (
+          isGetDocsUrlSelectorImported &&
+          !isGetShowMetabaseLinksSelectorImported &&
+          node?.callee?.type === "Identifier" &&
+          node?.callee?.name === "getDocsUrl"
+        ) {
+          context.report({
+            node,
+            message: ERROR_MESSAGE,
+          });
+        }
+
+        // call `getLearnUrl` selector
+        if (
+          isGetLearnUrlSelectorImported &&
+          !isGetShowMetabaseLinksSelectorImported &&
+          node?.callee?.type === "Identifier" &&
+          node?.callee?.name === "getLearnUrl"
+        ) {
+          context.report({
+            node,
+            message: ERROR_MESSAGE,
+          });
+        }
+
+        // call `MetabaseSettings.learnUrl` or `MetabaseSettings.docsUrl`
+        if (
+          isMetabaseSettingsImported &&
+          !isGetShowMetabaseLinksSelectorImported &&
+          node?.callee?.object?.type === "Identifier" &&
+          node?.callee?.object?.name === "MetabaseSettings" &&
+          ["learnUrl", "docsUrl"].includes(node?.callee?.property?.name)
+        ) {
+          context.report({
+            node,
+            message: ERROR_MESSAGE,
+          });
+        }
+      },
+      Literal(node) {
+        if (typeof node.value !== "string") {
+          return;
+        }
+
+        if (
+          LITERAL_METABASE_URL_REGEX.exec(node.value) &&
+          !isGetShowMetabaseLinksSelectorImported
+        ) {
+          context.report({
+            node,
+            message: ERROR_MESSAGE,
+          });
+        }
+      },
+      TemplateLiteral(node) {
+        const quasis = node.quasis;
```

Comment:
You can think of `quasis` as a parts of a template tag e.g.
```ts
const tag = `This is the first part ${1+1} this is the second part`
```

`quasis` would contain 2 elements
1. one with a value `This is the first part ` inside
2. and another with a value ` this is the second part` inside.

It seems the interpolations aren't included in `quasis`